pkg/ipc/ipc.go

```diff
@@ -403,6 +403,9 @@ func convertExtra(extraParts []flatrpc.CallInfo, dedupCover bool) *flatrpc.CallI
 		extra.Signal[i] = uint64(s)
 		i++
 	}
+	for _, part := range extraParts {
+		extra.Comps = append(extra.Comps, part.Comps...)
```

Comment:
Should we dedup them here?

Reply:
I'm not sure it's worth it -- now we anyway explicitly convert these to a `prog.CompMap` object when we want to actually substitute them, so they will be deduplicated anyway.

Reply:
Right, we dedup later.